src/Frontend/Core.hs

```diff
@@ -125,6 +129,8 @@ type instance CaptureData User               = AUID User
 type instance CaptureData IdeaJuryResultType = IdeaJuryResultType
 type instance CaptureData Role               = Role
 
+data PostResult a = UnsafePostResult
```

Comment:
ah, i see.  like this?: https://github.com/liqd/aula/pull/648/commits/e95c58c0bbc8222c6974e9f11241164664dfd6f5